Lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -1126,6 +1124,13 @@ namespace Js
 #endif
       isByteCodeDebugMode(false)
     {
+        if (nestedCount > 0)
+        {
+            NestedArray* nestedArray = RecyclerNewWithBarrierPlusZ(m_scriptContext->GetRecycler(),
+                nestedCount*sizeof(WriteBarrierPtr<FunctionProxy>), NestedArray, nestedCount);
+            SetAuxPtr(AuxPointerType::NestedArray, nestedArray);
```

Comment:
I was also hesitate to put this into Aux structure or directly attach to functionProxy.
from my statistic, 3686 out of 5811 functionBodys have m_nestedCount equals 0. I didn't measure FunctionProxy and ParseableFunctionInfo. let me to that measurement as well to make better decision. thanks!